<?xml version="1.0"?>
<launch>
	<node pkg="depth_laser_filter" type="max_range_filter_node" respawn="false" name="depth_laser_filter">
		<param name="min_th" value="0.19"/>
		<param name="laser_topic_in" value="scan"/>
		<param name="laser_topic_out" value="scan_filtered"/>
	</node>
</launch>
